webpack.config.js

```diff
@@ -21,16 +21,28 @@ module.exports = {
   },
   module: {
     rules: [
+      {
+        test: /\.modernizrrc.js$/,
+        use: ['@sect/modernizr-loader']
+      },
+      {
+        test: /\.modernizrrc(\.json)?$/,
+        use: [
+          '@sect/modernizr-loader','json-loader'
+        ],
+        type: 'javascript/auto'
+      },
       {
         test: /\.(js|ts)$/,
         include: [
           path.resolve(__dirname, 'node_modules/@hotwired/stimulus'),
           path.resolve(__dirname, 'node_modules/@stimulus/polyfills'),
+          path.resolve(__dirname, 'node_modules/@hotwired/turbo'),
```

Comment:
Don't we need `turbo-rails` here too?

Reply:
it doesn't give any error it doesn't look like it